samples/cellular/location/overlay-thingy91x-wifi-scan-only.conf

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
Do we need a copied Wi-Fi overlay config although name `overlay-nrf7002ek-wifi-scan-only.conf` is maybe not perfect?

Reply:
true, we could rename and use it for both.

Reply:
I've discussed about the name of the overlay config file and we feel it should have some reference to nrf wifi because it won't work with ESP or other chips. Our proposal is `overlay-nrf700x-wifi-scan-only.conf`.

Whatever renaming is done, `sample.yaml` has to be updated, and when we have the name fixed, I'll ask our test engineers to change the CI builds so that MOSH and positioning tests passes.

Reply:
> I've discussed about the name of the overlay config file and we feel it should have some reference to nrf wifi because it won't work with ESP or other chips. Our proposal is `overlay-nrf700x-wifi-scan-only.conf`.

Please do this renaming so we get the CI changes done also so you'll have one less issue getting it green.